O21.Game/Program.fs

```diff
@@ -32,80 +34,81 @@ let private runGame screenSize u95DataDirectory =
-                use stream = new MemoryStream(bytes)
-                use streamReader = new BinaryReader(stream, Encoding.UTF8, leaveOpen = true)
-                let header = SystemHeader.Load streamReader
-                printfn " - SystemHeader ok."
-            | "|FONT" ->
-                use stream = new MemoryStream(bytes)
-                use streamReader = new BinaryReader(stream, Encoding.UTF8, leaveOpen = true)
-                let fontFile = FontFile.Load streamReader
-                printfn " - Font ok."
-
-                for descriptor in fontFile.ReadDescriptors() do
-                    printfn $" - - Font descriptor: {descriptor.Attributes}"
-            | "|TOPIC" ->
-                use stream = new MemoryStream(bytes)
-                use streamReader = new BinaryReader(stream, Encoding.UTF8, leaveOpen = true)
-                let topic = TopicFile.Load streamReader
-                printfn " - Topic ok."
-
-                let mutable i = 0
-                for p in topic.ReadParagraphs() do
-                    printfn $" - Paragraph {p} ({p.DataLen1}, {p.DataLen2}) ok."
-
-                    let out1 = outputName + $"{i}.1"
-                    printfn $" - - Paragraph data: {out1}"
-                    File.WriteAllBytes(out1, p.ReadData1())
-
-                    let out2 = outputName + $"{i}.2"
-                    printfn $" - - Paragraph data: {out2}"
-                    File.WriteAllBytes(out2, p.ReadData2())
-
-                    if p.RecordType = ParagraphRecordType.TextRecord then
-                        let items = p.ReadItems(Encoding.GetEncoding 1251)
-                        printfn $"- - Items: {items.Settings}"
-                        for item in items.Items do
-                            printfn $"- - - {item}"
-
-                    i <- i + 1
-            | _ -> ()
-
-            exportImagesAsBmp outDir dibs
-    | [| "export"; inputFile; outDir |] ->
-        Async.RunSynchronously(async {
-            let! resources = Async.AwaitTask(NeExeFile.LoadResources inputFile)
-            exportImagesAsBmp outDir resources
+    
+    let reporter:CommandLineReporter = new CommandLineReporter()
+    
+    let matchArgs (command:BaseCommand) =
+        match command with
+        | :? StartGame as startCommand ->
+            if startCommand.screenSizes <> null then
+                let width, height = (startCommand.screenSizes[0], startCommand.screenSizes[1])
+                let size = struct(width, height)
+                runGame (Some size) startCommand.gameDirectory
+            else runGame None startCommand.gameDirectory
+        | :? ExportResources as exportCommand ->
+            Async.RunSynchronously(async {
+            let! resources = Async.AwaitTask(NeExeFile.LoadResources exportCommand.inputFilePath)
+            exportImagesAsBmp exportCommand.outputDirectory resources
         })
+        | :? HelpFile as helpCommand ->
+            Console.OutputEncoding <- Encoding.UTF8
```

Comment:
_(Minor)_ By the way, let's move this line to the initializer of `SystemConsole`?